src/blocks/tabs/Tabs.styled.ts

```diff
@@ -0,0 +1,119 @@
+import { Tabs as ReachTabs, TabList, Tab } from '@reach/tabs';
+import styled from 'styled-components';
+import { ModeProp } from '../Blocks.types';
+
+export const StyledFillTabs = styled(ReachTabs)`
+  display: flex;
+  flex-direction: column;
+  gap: var(--s4);
+`;
+
+export const StyledFillTabList = styled(TabList)<ModeProp>`
+  display: flex;
+  width: fit-content;
+  padding: var(--s1);
+  background-color: var(--${({ mode }) => (mode === 'light' ? 'gray-100' : 'gray-1000')});
```

Comment:
Can we use the `getBlocksColor` function here.
This looks very difficult to read and easy to make mistakes.

Example:
`${getBlocksColor(mode, { light: 'gray-600', dark: 'gray-300' })}`

Reply:
Done